### DISCUSSION TITLE
useRouter, useRoute is type any. does who know this problem?

### DISCUSSION THREAD

**Discussion by binochoi** [nuxt/nuxt#23062] [Questions]
![image](https://github.com/nuxt/nuxt/assets/59571263/a5afa9dc-94fa-4537-96c5-80ec7a7803d5)

How could this be?

The rest of the types are working fine.
It seems to be a pretty difficult situation.

**Comment by TheAlexLichter** [comment]
How to reproduce this? 🤔

**Reply by binochoi** [reply]
It was a problem of type conflict in my team code. It was not a matter of nuxt. Thank you for your interest!

**Reply by brandon-gs** [reply]
@binochoi what was the problem?

I have the same problem but I don't know what is wrong

**Reply by binochoi** [reply]
The problem occurred during tsconfig merging.

If not, Although it's a library of vue I remember having problems overwriting the router type itself on modules such as unplugin-vue-router once in a while.

if its happened suddenly i recommend that you do not trust latest version of nuxt.

recently there is a situation where I had to drop the version of nuxt and third party because of the weird problems that happened with the latest version.
